crates/bevy_ecs/src/system/query.rs

```diff
@@ -1075,23 +1177,17 @@ impl<'w, 's, Q: WorldQuery, F: WorldQuery> Query<'w, 's, Q, F> {
     /// # bevy_ecs::system::assert_is_system(player_system);
     /// ```
     ///
-    /// # Panics
+    /// # See also
     ///
-    /// Panics if the number of query results is not exactly one. Use
-    /// [`get_single`](Self::get_single) to return a `Result` instead of panicking.
+    /// - [`get_single`](Self::get_single) for the non-panicking version.
```

Comment:
These should probably link to-and-from their mutable twins?

Reply:
Yes, that was my intention for all the methods. Good catch 👍🏻